core/rio/api/src/test/resources/logback-test.xml

```diff
@@ -5,6 +5,9 @@
 			<pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %msg%n</pattern>
 		</encoder>
 	</appender>
+	<logger name="org.eclipse.rdf4j.rio">
+		<level value="warn"/>
```

Comment:
locally change to `debug` to see results of writing on stdout in several tests. Don't forget to change back before committing to avoid blowing up the build log.